PoGo.NecroBot.Logic/State/StateMachine.cs

```diff
@@ -29,6 +31,24 @@ public void SetFailureState(IState state)
             CancellationToken cancellationToken = default(CancellationToken))
         {
             var state = initialState;
+            var profilePath = Path.Combine(Directory.GetCurrentDirectory(), "");
+            var profileConfigPath = Path.Combine(profilePath, "config");
+
+            FileSystemWatcher configWatcher = new FileSystemWatcher();
+            configWatcher.Path = profileConfigPath;
+            configWatcher.Filter = "config.json";
+            configWatcher.NotifyFilter = NotifyFilters.LastWrite;
+            configWatcher.EnableRaisingEvents = true;
+            configWatcher.Changed += (sender, e) =>
+            {
+                if (e.ChangeType == WatcherChangeTypes.Changed)
+                {
+                    session.LogicSettings = new LogicSettings(GlobalSettings.Load(""));
+                    configWatcher.EnableRaisingEvents = !configWatcher.EnableRaisingEvents;
+                    configWatcher.EnableRaisingEvents = !configWatcher.EnableRaisingEvents;
```

Comment:
@khellang because event's data is coming twice(echo) so i need reset configWatcher

first EnableRaisingEvents  is false (stop watching)
second EnableRaisingEvents  is true (start watching)

so 2nd data doesn't come. i didnt found alternative or i dont now why data comes twice